pkg/router/template/util/util.go

```diff
@@ -0,0 +1,71 @@
+package util
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+
+	routeapi "github.com/openshift/origin/pkg/route/apis/route"
+)
+
+// Generate a regular expression to match route hosts (and paths if any).
+func GenerateRouteRegexp(hostname, path string, wildcard bool) string {
```

Comment:
godoc

Reply:
This was actually existing code that got moved. But I can add godoc to it.